frameworks/elastic/tests/test_shakedown.py

```diff
@@ -161,12 +172,12 @@ def test_plugin_install_and_uninstall(default_populated_index):
     config = sdk_marathon.get_config(FOLDERED_SERVICE_NAME)
     config['env']['TASKCFG_ALL_ELASTICSEARCH_PLUGINS'] = plugin_name
     sdk_marathon.update_app(FOLDERED_SERVICE_NAME, config)
-    check_plugin_installed(plugin_name, service_name=FOLDERED_SERVICE_NAME)
+    config.check_plugin_installed(plugin_name, service_name=FOLDERED_SERVICE_NAME)
```

Comment:
@loren Thanks for that catch! I'll just remove that renaming and live with the `from config import *` for the time being